roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -59,6 +59,28 @@ def _print_failure_summary(self):
                 indented = u'{}{}'.format(subsequent_indent, entry)
                 self._display.display(indented)
 
+        if self.__failures:
+            failed_checks = [
+                name
+                for name, result in failure['result']._result.get('checks', []).items()
+                for failure in self.__failures
+                if result.get('failed', False)
+            ]
+            # FIXME: get name of currently running playbook, if possible.
+            NAME_OF_PLAYBOOK = 'playbooks/byo/config.yml'
```

Comment:
This may be tricky to figure out, if possible. Didn't have much time to keep digging. We may also just drop this information from the output.